HLTrigger/Configuration/python/HLT_75e33/sequences/hgcalLocalRecoSequence_cfi.py

```diff
@@ -6,5 +6,33 @@
 from ..modules.hgcalMergeLayerClusters_cfi import *
 from ..modules.HGCalRecHit_cfi import *
 from ..modules.HGCalUncalibRecHit_cfi import *
+# Heterogeneous HGCAL EE layer clusters
+from ..modules.hltHgcalSoARecHitsProducer_cfi import *
+from ..modules.hltHgcalSoARecHitsLayerClustersProducer_cfi import *
+from ..modules.hltHgcalSoALayerClustersProducer_cfi import *
+from ..modules.hltHgcalLayerClustersFromSoAProducer_cfi import *
 
-hgcalLocalRecoSequence = cms.Sequence(HGCalUncalibRecHit+HGCalRecHit+hgcalLayerClustersEE+hgcalLayerClustersHSci+hgcalLayerClustersHSi+hgcalMergeLayerClusters)
+hgcalLocalRecoSequence = cms.Sequence(
+        HGCalUncalibRecHit+
+        HGCalRecHit+
+        hgcalLayerClustersEE+
+        hgcalLayerClustersHSci+
+        hgcalLayerClustersHSi+
+        hgcalMergeLayerClusters)
+
+_hgcalLocalRecoSequence_heterogeneous = cms.Sequence(
+        HGCalUncalibRecHit+
+        HGCalRecHit+
+        hltHgcalSoARecHitsProducer+
+        hltHgcalSoARecHitsLayerClustersProducer+
+        hltHgcalSoALayerClustersProducer+
+        hltHgCalLayerClustersFromSoAProducer+
+        hgcalLayerClustersHSci+
+        hgcalLayerClustersHSi+
+        hgcalMergeLayerClusters)
+
+from Configuration.ProcessModifiers.heterogeneousCLUE_cff import heterogeneousCLUE
+heterogeneousCLUE.toReplaceWith(hgcalLocalRecoSequence, _hgcalLocalRecoSequence_heterogeneous)
+heterogeneousCLUE.toModify(hgcalMergeLayerClusters,
```

Comment:
I would prefer to use directly the `alpaka` modifier, instead of introducing a new one.

Or to make the alpaka version the default one.